src/transceivercontroller/SimulcastTransceiverController.ts

```diff
@@ -98,11 +98,17 @@ export default class SimulcastTransceiverController extends DefaultTransceiverCo
         direction: 'inactive',
         streams: [this.defaultMediaStream],
       });
+
+      /* istanbul ignore next */
+      if (this.meetingSessionContext?.audioProfile.hasRedundancyEnabled()) {
+        // This will perform additional necessary setup for the audio transceiver.
+        this.setupAudioRedWorker();
+      }
     }
 
     if (!this._localCameraTransceiver) {
       const encodingParams = Array.from(this.videoQualityControlParameterMap.values());
-      this._localCameraTransceiver = this.peer.addTransceiver('video', {
+      this._localCameraTransceiver = this.setupVideoTransceiver({
```

Comment:
Hmm, I dont understand the need for this. Id `this.peer.addTransceiver` not working?

Reply:
The `setupVideoTransceiver` does some additional setup after adding the video transceiver to the peer connection. I also sets up pass through transforms if RED is enabled, which allows video to be sent/received. This is actually an issue with [WebRTC Encoded Transform](https://www.w3.org/TR/webrtc-encoded-transform/) where enabling this feature makes it so that every single transceiver requires a transform or no media send/receive will work on that transceiver.